lib/ap/plan9/signal.c

```diff
@@ -64,7 +64,10 @@ void
 	_sighdlr[sig] = func;
 	return oldf;
 }
-
+/*
+ * Wrong, for learning purposes.
+ * Real is in lib/ap/amd64 directory
+ *
 int
 sigsetjmp(sigjmp_buf buf, int savemask)
```

Comment:
The code and similar comment comes from old APE, but you're in right. It made me fail and I don't know about a libc out there which could have sigsetjmp implemented in C because different compilers doesn't handle at all the stack and regs for this signal. I'll remove it.